packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts

```diff
@@ -172,67 +199,134 @@ export class CucumberJSAllureFormatter extends Formatter {
       this.currentTest.addLabel(LabelName.TAG, tag.name);
 
       for (const label in this.labels) {
-        if (!this.labels[label]) continue;
+        if (!this.labels[label]) {
+          continue;
+        }
         for (const reg of this.labels[label]) {
           const match = tag.name.match(reg);
           if (match != null && match.length > 1) {
             this.currentTest.addLabel(label, match[1]);
           }
         }
       }
+
+      for (const link in this.links) {
+        if (!this.links[link]) {
+          continue;
+        }
+        for (const reg of this.links[link]) {
+          const match = tag.name.match(reg);
+          if (match != null && match.length > 1) {
+            if (link === LinkType.ISSUE && this.issuesTrackerUrlPattern) {
```

Comment:
How could I create a link with custom type? Maybe add url template to `links` configuration section? Something like

```js
links: {
  issue: {
     pattern: [/@issue:(.*)/]
     urlTemplate: "http://localhost:8080/issue/%s"
  },
  tms: {
     pattern: [/@tms:(.*)/]
     urlTemplate: "http://localhost:8080/tms/%s"
  }
},
```

Reply:
ok!

Reply:
@baev Sorry for the delay, I've been very busy these last few days... check the new code, please.